tests/phpunit/Writer/MSSQLTest.php

```diff
@@ -22,6 +17,9 @@ class MSSQLTest extends BaseTest
 
     private $config;
 
+    /**
```

Comment:
Tohle bych asi nepřidával. V tom scénáři, co se testuje, by to nikdy hodit nemělo.